test/e2e/tests/hardware-wallets/ledger/ledger-erc721.spec.ts

```diff
@@ -49,8 +51,47 @@ describe('Ledger Hardware', function (this: Suite) {
         await testDappPage.checkERC721TokenAddressesValue(
           '0xcB17707e0623251182A654BEdaE16429C78A7424',
         );
+      },
+    );
+  });
+  it('mints an ERC-721 token', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder()
+          .withLedgerAccount()
+          .withPermissionControllerConnectedToTestDapp({
+            account: KNOWN_PUBLIC_KEY_ADDRESSES[0].address,
+          })
+          .build(),
+        title: this.test?.fullTitle(),
+        dapp: true,
+        smartContract: [
+          {
+            name: erc721,
+            deployerOptions: {
+              fromAddress: KNOWN_PUBLIC_KEY_ADDRESSES[0].address,
+            },
```

Comment:
whit this option, we can set the deployer address to our hardware wallet account, and then perform subsequent contract interactions, as we are the owner of the contract